src/webpack/rules/index.js

```diff
@@ -0,0 +1,21 @@
+import loadJavascript from './loadJavascript'
+import loadCSS from './loadCSS'
+import loadAnything from './loadAnything'
```

Comment:
You did not import and export ``` loadFonts.js ```.

Reply:
Because every file that is not supposed to be compiled will be processed through `loadAnything.js` it would be redundant to include a separate loader doing the same thing. In case you want to modify the loader toolchain I included the `loadFonts.js' so you can swap the `loadAnything.js` with it. That would allow you to configure loaders for every file-type not handled by the provided loaders 'loadJavascript.js`, 'loadCSS.js` and `loadFonts.js`.
Maybe there will be an missing `loadFiles.js` that could handle only images, videos, etc. but I think  it wouldn't be a hazzle to write every other loaders yourself in case you want a custom toolchain.

Perhaps, it is a good idea to describe this behavior more precisely in the `README.md`

Reply:
### Sounds great.